src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -2018,6 +2018,30 @@ private void PerformSchedulingActions(IEnumerable<ScheduleResponse> responses)
             }
         }
 
+        internal int CreateRarNode()
+        {
+            string nodeLocation = _buildParameters?.NodeExeLocation ?? BuildEnvironmentHelper.Instance.CurrentMSBuildExePath;
+            if (string.IsNullOrEmpty(nodeLocation))
+            {
+                // Couldn't find a path to MSBuild.exe; can't create a new node.
+                return -1;
+            }
+
+            bool nodeReuse = _buildParameters?.EnableNodeReuse ?? true;
+            bool lowPriority = _buildParameters?.LowPriority ?? false;
+            string commandLineArgs = $"/nologo /nodemode:3 /nodeReuse:{nodeReuse} /low:{lowPriority}";
+            try
+            {
+                int nodeId = NodeProviderOutOfProcBase.LaunchNode(nodeLocation, commandLineArgs);
+                return nodeId;
+            }
+            catch (Exception)
+            {
```

Comment:
minor:  is exception swallowing a pattern in the code base?  Would there be value added, logging the exception?

Reply:
Remove exception handling from here and let it do in task, where is logger which can do the logging.